graylog2-web-interface/src/stores/permissions/EntityShareStore.js

```diff
@@ -53,24 +52,25 @@ const EntityShareStore: EntityShareStoreType = singletonStore(
     },
 
     searchPaginatedUserShares(username: string, page: number, perPage: number, query: string, additionalQueries?: AdditionalQueries): Promise<PaginatedEnititySharesType> {
-      // const url = PaginationURL(ApiRoutes.EntityShareController.userSharesPaginated(username).url, page, perPage, query, additionalQueries);
-      // const promise = fetch('GET', qualifyUrl(url)).then((response: PaginatedUserSharesResponse) => {
-      //   return {
-      //     list: Immutable.List(response.entities.map((se) => SharedEntity.fromJSON(se))),
-      //     context: {
-      //       userCapabilities: response.context.user_capabilities,
-      //     },
-      //     pagination: {
-      //       count: response.count,
-      //       total: response.total,
-      //       page: response.page,
-      //       perPage: response.per_page,
-      //       query: response.query,
-      //     },
-      //   };
-      // });
-
-      const promise = permissionsMock.searchPaginatedEntitySharesResponse(page, perPage, query, additionalQueries);
+      const url = PaginationURL(ApiRoutes.EntityShareController.userSharesPaginated(username).url, page, perPage, query, additionalQueries);
+      const promise = fetch('GET', qualifyUrl(url)).then((response) => {
+        return {
+          list: Immutable.List(response.entities.map((se) => SharedEntity.fromJSON(se))),
+          context: {
+            granteeCapabilities: response.context.grantee_capabilities,
+          },
+          pagination: {
+            additionalQueries: response.additional_queries,
+            count: response.count,
+            total: response.total,
+            page: response.page,
+            perPage: response.per_page,
+            query: response.query,
+          },
+        };
+      });
+
+      // const promise = permissionsMock.searchPaginatedEntitySharesResponse(page, perPage, query, additionalQueries);
```

Comment:
I just saw, I forgot to remove this line. If there is nothing else that needs to be adjusted, I will remove it in `feature/permissions`